src/SFA.DAS.Reservations.Api/StartupExtensions/NServiceBusStartUp.cs

```diff
@@ -20,6 +20,7 @@ public static class NServiceBusStartUp
         {
             var endpointConfiguration = new EndpointConfiguration("SFA.DAS.Reservations.Api")
                 .UseInstallers()
+                .UseErrorQueue("SFA.DAS.Reservations.Api-errors")
```

Comment:
could have local const EndPointName = "SFA.DAS.Reservations.Api" and concat "-errors" on the end for the error queue